### PR TITLE
Fix search for uppercase emails

### PR DESCRIPTION
**Pull request by nikulin**
Try 
python -m prpr.main -m all --student shasall@yandex.ru

### CODE REVIEW

File: prpr/filters.py [hunk 1]
```diff
@@ -80,7 +80,8 @@ def filter_homeworks(
     if problems:
         result = [h for h in result if h.problem in problems]
     if student:
-        result = [h for h in result if student in h.student]
+        search_string = student.lower()
+        result = [h for h in result if search_string in h.student.lower()]
```
Comment (salmiakki) [inline prpr/filters.py:84]:
```suggestion
        result = [h for h in result if student.lower() in h.student.lower()]
```

Comment (salmiakki) [reply prpr/filters.py:84]: Let's inline the search string.

Comment (salmiakki) [reply prpr/filters.py:84]: Yuri, thanks for the suggestion! Could you please update the change log accordingly?

**Comment by salmiakki** [comment]
Yuri, thanks for the suggestion! 👍

Could you please update the change log accordingly?

File: prpr/filters.py (hunk 1 above)
Comment (nikulin) [reply prpr/filters.py]: Done.

**Comment by salmiakki** [comment]
Yuri, please update the change log as well.

**Comment by nikulin** [comment]
> Yuri, please update the change log as well.
Done. Also tests were added.

**Pull request merged**